packages/webapp-server/package.json

```diff
@@ -4,19 +4,23 @@
   "main": "src/index.ts",
   "license": "MIT",
   "scripts": {
-    "dev": "ts-node-dev src/index.ts",
+    "dev": "nodemon src/index.ts",
     "start": "ts-node src/index.js",
+    "dev:docker": "docker compose down && docker compose up",
```

Comment:
Running the dev:docker command will spin up both the app and a mongodb image. Just need to add a webapp-server/.env with the variables MONGODB_USER and MONGODB_PASSWORD set to anything you want.